src/viz/viz.jl

```diff
@@ -12,8 +12,8 @@ for existing functions/methods.
 function explore() end
 
 # Scenario plotting methods
-function scenario() end
-function scenario!() end
+function scenarios() end
+function scenarios!() end
```

Comment:
```suggestion
function scenarios() end
function scenarios!() end

function scenario(args...; kwargs...)
    @warn """
    viz.scenario() is now deprecated and will be removed in ADRIA v1.0
    
    Instead, use:
        `viz.scenarios()`
    """
    
    return scenarios(args...; kwargs...)
end
```

Reply:
Yes, it works.

![image](https://github.com/open-AIMS/ADRIA.jl/assets/8040719/7969cc35-50b9-4f7b-8dac-3d93f1df2140)